app/views/books/_book.html.erb

```diff
@@ -1,6 +1,13 @@
 <tr>
   <td class="px-3"><%= book.name %></td>
   <td class="px-3"><%= book.isbn %></td>
+  <td>
+    <% if book.cover.attached? %>
+      <%= image_tag(book.cover, class: "m-2 img-thumbnail", width: "150") %>
+    <% else %>
+      <%= image_tag("sample-book-cover.jpg", class: "m-2 img-thumbnail", width: "150") %>
```

Comment:
Good call 👍 

If you want to reduce the duplication of the `image_tag` and its arguments, you could do the following:

```ruby
<% book_cover = book.cover.attached? ? book.cover : "sample-book-cover.jpg" %>
<%= image_tag(book_cover, class: "m-2 img-thumbnail", width: "150") %>
```